R/fn.R

```diff
@@ -392,17 +392,25 @@ fn_env <- function(fn) {
 #'   If a **function**, it is used as is.
 #'
 #'   If a **formula**, e.g. `~ .x + 2`, it is converted to a function
-#'   with two arguments, `.x` or `.` and `.y`. This allows you to
+#'   with up to two arguments: `.x` (single argument) or `.x` and `.y`
+#'   (two arguments). The `.` placeholder can be used instead of `.x`.
+#'   This allows you to
```

Comment:
I did this awful line break to make the diff smaller. Also, I think you wrap at a different column than I do.

Reply:
I think it's ok to create a larger diff because of rewrapping. I usually wrap at 72 because this way two pages fit nicely on 13in laptop screens with a legible font size.

No worries though.